src/operator/quantization/quantize_graph_pass.cc

```diff
@@ -156,29 +158,54 @@ Graph QuantizeGraph(Graph &&src) {
           new_node->inputs.emplace_back(mirror_entry);
         } else if (!NeedQuantize(e.node, excluded_nodes) &&
                    (mirror_node->op() == nullptr ||
-                    mirror_node->op()->name != "_contrib_quantize")) {
+                    mirror_node->op()->name != "_contrib_quantize") &&
```

Comment:
Make _contrib_quantize a constant variable. Or even better this should be a function that returns op name

Reply:
This is the convention naming in current MXNet, right?
For this case, the string name here is already interpretable as either a constant variable or a function.

Reply:
Comparing to a plain string is error prone. What if someone mis-spelled by one char. It's going to be hard to debug.
In fact, I think a more rigorous way to check if an operator may be checking the operator itself: if (op ==  nnvm::Op::Get("_contrib_quantize"))

Reply:
Yes, that will be better and more strict way compared with original one.
I'm thinking  about checking `entry` (like: `!mirror_entry_map.count(e)`) but not `mirror_node` here, since we stores the mapping between current entries and the entries outgoing from `_contrib_quantize` node via  `nnvm::NodeEntryMap<NodeEntry>` for both single output and multiple outputs. `mirror_node` can handle single output but not multiple outputs, while `entry` can handle both cases.
What do you think about this?

Reply:
Checking `entry` as you suggested is fine. Checking op name with a string seems to be error-prone and requires many updates if we later change the op name